drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -1919,15 +1933,38 @@ function DragAndDropBlock(runtime, element, configuration) {
     var isPastDue = function () {
         return !configuration.has_deadline_passed;
     };
-
-    var canShowAnswer = function() {
-        return configuration.mode === DragAndDropBlock.ASSESSMENT_MODE && !attemptsRemain();
+    var isAttempted = function () {
+        return state.attempts > 0;
+    };
+    var isCorrect = function () {
+        return state.last_action_correct;
+    };
+    var isClosed = function () {
+        if(isPastDue() || !attemptsRemain()) return true;
+        return false;
     };
 
     var attemptsRemain = function() {
         return !configuration.max_attempts || configuration.max_attempts > state.attempts;
     };
 
+
+    var canShowAnswer = function() {
+        if(configuration.mode !== DragAndDropBlock.ASSESSMENT_MODE) return false;
+        else if (configuration.show_answer_status === SHOW_ANSWER_STATUSES.NEVER) return false;
```

Comment:
I don't see the `show_answer_status` being passed as a configuration value [in the student_view](https://github.com/openedx/xblock-drag-and-drop-v2/blob/master/drag_and_drop_v2/drag_and_drop_v2.py#L336), so this will always return false as the `show_answer_status` will be `undefined`.

I think the student_view's configuration should have something like
```
"show_answer_status": self.showanswer
```

Reply:
@abodacs I was testing the changes now, and I don't see the above changes (specifically updates to `drag_and_drop.js`) in the PR anymore. Did some commits go missing while rebasing?